config/webpack.config.base.js

```diff
@@ -62,7 +63,7 @@ const MEDIA_URL_LOADER = {
 };
 
 const CSS_LOADER = {
-  loader: 'style!css',
+  loader: ExtractTextPlugin.extract('css?modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]'),
```

Comment:
are you sure you're using this correctly? this looks different from what I see in the docs:

https://github.com/webpack/extract-text-webpack-plugin